### DISCUSSION TITLE
Fast tracks page can not be reached

### DISCUSSION THREAD

**Discussion by lanming2023** [owid/owid-grapher#2407] [Q&A]
Do I need additional set up in order to access Fast-track page ? right now , if i click fast-track tab in the admin site, it redirects me to this url 'http://owid-analytics:8083/' and showed site cannot be reached. 

![image](https://github.com/owid/owid-grapher/assets/138816872/f541ea1a-9ad1-4a2b-bc00-da0236e900a5)

![image](https://github.com/owid/owid-grapher/assets/138816872/384db851-337c-4bc8-af8d-922b5907be57)

**Comment by danyx23** [comment]
Hi there! fasttrack is a separate tool written in python that we run in a vpn so it's not useable for non-staff or on a local dev machine.

**Comment by lanming2023** [comment]
*Thanks for your reply, Daniel*

*Xiaoh**ua Cai  |  **Developer*
*Email:* ***@***.***
*P**ronouns: *she/her/hers
w <http://www.affinitybridge.com/>ww.a <http://www.affinitybridge.com>
ffinitybri <http://www.affinitybridge.com>dge.com
<http://www.affinitybridge.com>
 I respectfully acknowledge that I live and work on unceded and traditional
territories of the Coast Salish, səl̓ilwətaɁɬ təməxʷ (Tsleil-Waututh),
Skwxwú7mesh-ulh Temíx̱w (Squamish), S’ólh Téméxw (Stó:lō), Stz'uminus, and
xʷməθkʷəy̓əm Nations.
